en/flight_controller/snapdragon_flight_advanced.md

```diff
@@ -249,6 +249,15 @@ If it still doesn't work, check [this answer on StackOverflow](http://askubuntu.
 
 ### Board doesn't start / is boot-looping / is bricked
```

Comment:
We use first letter capitalisation on all headings - i.e. Board Doesn't Start / Board Boot-Looping / Board Bricked

(change everywhere)

Reply:
@hamishwillee Ok, are there any exceptions about this? I went through the guide and found several instances where this is not the case. Is is 'Forums and Chat' or 'Forums And Chat'?

Reply:
Yes, you don't capitalise the small connecting words unless they are at the beginning of the sentence - and, on, the, with, for, is etc. 

With respect to the license, perhaps we could simply add a note.

> **Note** This section of text originates from the [ATLFlightDocs](https://github.com/ATLFlight/ATLFlightDocs#snapdragon-flight-documentation)

There are a couple of other minor comments. Assuming you're happy with the above I can merge this and make the changes as needed (?)